projects/github-actions/repo-gardening/src/tasks/triage-issues/ai-labeling.js

```diff
@@ -94,6 +94,27 @@ Example response format:
 	return { labels, explanations: parsedResponse };
 }
 
+/**
+ * Clean up the issue content for OpenAI processing.
+ * Remove links from the issue content.
+ *
+ * @param {string} content - Issue body content.
+ *
+ * @return {string} Cleaned up issue content.
+ */
+function cleanIssueContent( content ) {
+	// Remove links in the format [link text](url)
+	content = content.replace( /\[.*?\]\(.*?\)/g, '' );
```

Comment:
Probably you'd want to keep the link text, just not the url?

Reply:
That's a good point, I didn't think of that. What do you think of 788d7abc98e07998a766501dc24ca783cd027bca?